taskweaver/module/tracing.py

```diff
@@ -0,0 +1,156 @@
+from typing import Literal, Optional
+
+from injector import inject
+
+from taskweaver.config.module_config import ModuleConfig
+
+
+class TracingConfig(ModuleConfig):
+    def _configure(self):
+        self._set_name("tracing")
+        self.enabled = self._get_bool("enabled", False)
+        self.endpoint = self._get_str("endpoint", "http://127.0.0.1:4318/v1/traces")
+        self.service_name = self._get_str("service_name", "taskweaver.opentelemetry.tracer")
```

Comment:
feel like the defult name is too long when displaying the Jaeger UI (component name is then difficult to read), may consider simplify it

Reply:
need to have both 'taskweaver' and 'opentelemetry'.

Reply:
for example, using "tw" and "otel" are not allowed? in Jaeger, its readability is not very good.

Reply:
taskweaver.optl.tracer? tw is not so good :(

Reply:
okay, taskweaver.otel.tracer? otel is the official abbreviation for OpenTelemetry

Reply:
OpenTelemetry Protocol (OTLP)